README.md

```diff
@@ -24,68 +27,125 @@ We like to think of it as `kubectl` for clusters.
   </a>
 </p>
 
-## Launching a Kubernetes cluster hosted on AWS
+
+## Launching a Kubernetes cluster hosted on AWS or GCE
 
 To replicate the above demo, check out our [tutorial](/docs/aws.md) for
 launching a Kubernetes cluster hosted on AWS.
 
+To install a Kubernetes cluster on GCE please follow this [guide](/docs/tutorial/gce.md).
+
 
 ## Features
 
-* Automates the provisioning of Kubernetes clusters in ([AWS](/docs/aws.md))
+* Automates the provisioning of Kubernetes clusters in [AWS](/docs/aws.md) and [GCE](/docs/tutorial/gce.md)
 * Deploys Highly Available (HA) Kubernetes Masters
-* Supports upgrading from [kube-up](/docs/upgrade_from_kubeup.md)
 * Built on a state-sync model for **dry-runs** and automatic **idempotency**
-* Ability to generate configuration files for AWS [CloudFormation](https://aws.amazon.com/cloudformation/) and Terraform [Terraform configuration](/docs/terraform.md)
+* Ability to generate [Terraform](/docs/terraform.md)
 * Supports custom Kubernetes [add-ons](/docs/addons.md)
 * Command line [autocompletion](/docs/cli/kops_completion.md)
-* Manifest Based API [Configuration](/docs/manifests_and_customizing_via_api.md)
-* Community supported!
+* YAML Manifest Based API [Configuration](/docs/manifests_and_customizing_via_api.md)
+* [Templating](/docs/cluster_template.md) and dry-run modes for creating
+ Manifests
+* Choose from eight different CNI [Networking](/docs/networking) providers out-of-the-box
+* Supports upgrading from [kube-up](/docs/upgrade_from_kubeup.md)
+* Capability to add containers, as hooks, and files to nodes via cluster manifest
 
 
-## Documentations
+## Documentation
 
 Documentation is in the `/docs` directory, [and the index is here.](docs/README.md)
 
 
+## Kubernetes Release Compatibility
+
+
+### Kubernetes Version Support
+
+kops is intended to be backward compatible.  It is always recommended to use the
+latest version of kops with whatever version of Kubernetes you are using. One
+exception, kops supports the equivalent Kubernetes minor release number.   A
+minor version is the second digit in the release number.  kops version 1.8.0
+has a minor version of 8. The numbering follows the semantic versioning
+specification, MAJOR.MINOR.PATCH.
+
+For example kops, 1.8.0 does not support Kubernetes 1.9.2, but kops 1.9.0
+supports Kubernetes 1.9.2 and previous Kubernetes versions. Only when kops minor
+version matches, the Kubernetes minor version does kops officially support the
+Kubernetes release.  kops does not stop a user from installing mismatching
+versions of K8s, but Kubernetes releases always require kops to install specific
+versions of components like docker, that tested against the particular
+Kuberenetes version.
```

Comment:
they're breeding bro!

Reply:
This is helpful but I almost think we need a diagram or a matrix in addition. Does not need to be a part of this PR

Reply:
Never said I could spell.  I am going to add a table matix here